tower/RetractTower.scad

```diff
@@ -0,0 +1,82 @@
+// Retract Tower
```

Comment:
Calibration tests are not necessary to test a slicer. I consider this out of scope.